changelog/fragments/upgrade-operator-framework-api.yaml

```diff
@@ -0,0 +1,19 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Upgrade operator-framework/api version from `v0.3.13` to `0.3.20`. With this upgrade:
+      1. Users can specify the path to a json-encoded file contating custom categories through an environment
+          variable while performing optional operatorHub UI validation for CSV categories. Refer to [api #69](https://github.com/operator-framework/api/pull/69).
+      2. Operator bundle contains `properties` field for dependency resolution. Refer to [api #66](https://github.com/operator-framework/api/pull/66).
+      3. Webhook container port specified in CSV defaults to `443` if not specified by the user. Refer to [api #71](https://github.com/operator-framework/api/pull/71).
```

Comment:
IMO we need an entry for each one and we should such as we did for the kb upgrade. See; https://github.com/operator-framework/operator-sdk/blob/master/changelog/fragments/upgrade-kb.yaml. The 3 is not a bugfix?

Reply:
I agree with @camilamacedo86 that it would be better to split into multiple entries (which can all be in this file; `entries` is a list). The `description` is meant to be a single line in the changelog. As is, this will render all on one line in the changelog because of the [folded style `>` YAML indicator](https://yaml.org/spec/1.2/spec.html#id2796251).